### PR TITLE
changed to non-holonomic robot (both in robot model and movebase)

### PR DESCRIPTION
**Pull request by marc-hanheide**
this changes everything to use a non-holonomic base. Much more realistic for our robot. It uses the MotionVW actuator and sets all yaml configs to be aware of this restriction. Tested on my machine.

### CODE REVIEW

File: strands_morse_2dnav/strands_morse_move_base/dwa_planner_ros.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 DWAPlannerROS:
   acc_lim_th: 2.0
   acc_lim_x: 1.0
-  acc_lim_y: 1.0
+  acc_lim_y: 0.0
```
Comment (marc-hanheide) [inline strands_morse_2dnav/strands_morse_move_base/dwa_planner_ros.yaml:4]: Is this the right way to let DWA know we can't move to the side??? Is there a more elegent way

File: strands_sim/src/strands_sim/builder/robots/scitosA5.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def __init__(self, debug = True):
         ###################################
 
         # Motion control
-        self.motion = MotionXYW()
+        self.motion = MotionVW()
```
Comment (marc-hanheide) [inline strands_sim/src/strands_sim/builder/robots/scitosA5.py:23]: I also tried MotionVWDiff(), but that one crashes...

File: strands_morse_2dnav/strands_morse_move_base/dwa_planner_ros.yaml (hunk 1 above)
Comment (bfalacerda) [inline strands_morse_2dnav/strands_morse_move_base/dwa_planner_ros.yaml:4]: Don't know of a better way... We were trying just with holonomic_robot: false but the planner tries to move the robot on y anyway, so as of now this change is needed

Comment (marc-hanheide) [inline strands_morse_2dnav/strands_morse_move_base/dwa_planner_ros.yaml:4]: So, does this work for you then? Then I'd merge it in.

Marc

File: strands_sim/src/strands_sim/builder/robots/scitosA5.py (hunk 2 above)
Comment (bfalacerda) [inline strands_sim/src/strands_sim/builder/robots/scitosA5.py:23]: MotionVWDiff() requires a WheeledRobot morse model (our robot is a "general" robot now), which in turns requires adding wheels in the blender model for the robot. This is not done right now

File: strands_morse_2dnav/strands_morse_move_base/dwa_planner_ros.yaml (hunk 1 above)
Comment (bfalacerda) [inline strands_morse_2dnav/strands_morse_move_base/dwa_planner_ros.yaml:4]: Yes, we tested it and it's working

**Comment by marc-hanheide** [comment]
I now merge this myself after confirmation by @BFALacerda

**Pull request merged**